studio.py

```diff
@@ -1,10 +1,11 @@
+from __future__ import absolute_import, print_function
```

Comment:
Is this necessary? I notice you've used relative imports elsewhere. Can we be consistent throughout. I'm not sure I have a preference between relative and absolute.

Reply:
Removed